scripts/run_integration_tests_aarch64.sh

```diff
@@ -30,13 +30,14 @@ build_spdk_nvme() {
     cp "$WORKLOADS_DIR/spdk/build/bin/nvmf_tgt" $SPDK_DEPLOY_DIR/nvmf_tgt
     cp "$WORKLOADS_DIR/spdk/scripts/rpc.py" $SPDK_DEPLOY_DIR/rpc.py
     cp -r "$WORKLOADS_DIR/spdk/scripts/rpc" $SPDK_DEPLOY_DIR/rpc
+    cp -r "$WORKLOADS_DIR/spdk/python" $SPDK_DEPLOY_DIR/../
 }
 
 build_virtiofsd() {
     VIRTIOFSD_DIR="$WORKLOADS_DIR/virtiofsd_build"
     VIRTIOFSD_REPO="https://gitlab.com/virtio-fs/virtiofsd.git"
 
-    checkout_repo "$VIRTIOFSD_DIR" "$VIRTIOFSD_REPO" v1.1.0 "220405d7a2606c92636d31992b5cb3036a41047b"
+    checkout_repo "$VIRTIOFSD_DIR" "$VIRTIOFSD_REPO" v1.6.1 "7886ab7c819fd8de370d5a3bbfaae44de56923ed"
```

Comment:
Is this related?

Reply:
@michael2012z 
If I use the old version of virtiofsd
```
Error running ch-remote command: "target/aarch64-unknown-linux-gnu/release/ch-remote" "--api-socket" "/tmp/ch40QpPE/cloud-hypervisor.sock" "resume"
stderr: Error running command: Server responded with an error: InternalServerError: ApiError(VmResume(Resume(Resume(Invalid transition: InvalidStateTransition(Running, Running)))))

failures:
    common_parallel::test_virtio_fs
    common_parallel::test_virtio_fs_hotplug
    common_parallel::test_virtio_fs_multi_segment                                                        
    common_parallel::test_virtio_fs_multi_segment_hotplug
```